### ISSUE TITLE
NineManga: Chapter images do not load

### ISSUE THREAD

**Issue by KasukeLp23** [tachiyomiorg/tachiyomi-extensions#12308]
### Source information

NineManga 1.2.16

### Steps to reproduce

1. Open extension
2. Open manga
3. Open chapter

### Expected behavior

The manga images appear normally

### Actual behavior

The images do not appear, the error "Could not load the image" appears. (Image 1)

### Tachiyomi version

0.13.4

### Android version

Android 7

### Other details

**Note:** _I have cleaned cookies and still have the problem._

**This happens with any extension manga**
![Screenshot_20220623-084809](https://user-images.githubusercontent.com/70596255/175317530-31c1e70e-009e-4d5f-aed2-ddfb67955252.jpg)

**In the webview it works without problems, the images of the chapters appearing**
**Image removed due to containing NSFW**

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.4](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by alessandrojean** [comment]
Please do not post NSFW images on GitHub.

**Comment by Riztard** [comment]
only happen to "ES"

**Comment by SpectreOfDark** [comment]
Just today I am getting this error, no image is loaded in the app, and only in "NinemangaEs (ES)", I tried the "NinemangaEn (EN)" and if it worked.
I have also tested both in WebView and in the browser and if they load the images.
Sometimes after entering the WebView, in some manga if the images are loaded, but when emptying the cache they all go wrong again.
The version of "Ninemanga" is 1.2.16
The version of tachiyomi I use is 0.13.6
And my android version is 12
